src/browser/browser.ts

```diff
@@ -230,10 +248,27 @@ export class Browser {
       this.log.debug('Taking screenshot', 'filePath', options.filePath);
     }
 
+    if (options.fullPageImage) {
+      this.log.debug('TODO: take full screen image', 'url', options.url);
```

Comment:
@ArturWierzbicki -- how involved was this process?  If easy to port, can we try that?